docs/install/install-knative-gcp.md

```diff
@@ -79,26 +79,25 @@ ko apply -f ./config
 1. Pick a knative-gcp release version:
 
    ```shell
-   export KGCP_VERSION=v0.15.0
+   export KGCP_VERSION=v0.19.0
    ```
 
-1. First install the CRDs by running the `kubectl apply` command with the
-   `--selector` flag. This prevents race conditions during the install, which
+1. First install the CRDs by running the `kubectl apply` for `cloud-run-events-pre-install-jobs.yaml`.
+This prevents race conditions during the installation, which
    cause intermittent errors:
 
    ```shell
-   kubectl apply --selector messaging.cloud.google.com/crd-install=true \
-   --filename https://github.com/google/knative-gcp/releases/download/${KGCP_VERSION}/cloud-run-events.yaml
-   kubectl apply --selector events.cloud.google.com/crd-install=true \
-   --filename https://github.com/google/knative-gcp/releases/download/${KGCP_VERSION}/cloud-run-events.yaml
+   kubectl apply --filename https://github.com/google/knative-gcp/releases/download/${KGCP_VERSION}/cloud-run-events-pre-install-jobs.yaml
    ```
 
-1. To complete the install run the `kubectl apply` command again, this time
-   without the `--selector` flags:
+1. To complete the installation, run the `kubectl apply` for `cloud-run-events.yaml`:
 
    ```shell
    kubectl apply --filename https://github.com/google/knative-gcp/releases/download/${KGCP_VERSION}/cloud-run-events.yaml
    ```
+**Notes:** For different releases, the installation files may be different (e.g. some old releases don't have `cloud-run-events-pre-install-jobs.yaml`).
```

Comment:
So the old instruction still works for old releases, right?

Reply:
I made a mistake before. We are still using `events.cloud.google.com/crd-install=true` so I added back this command in latest commit. But we don't use `messaging.cloud.google.com/crd-install=true`, even I went back to v0.12.0, this label is not used.

Reply:
nit: I think it might be easier to just create another file such as `install-knative-gcp-up-to-v0-18.md` which contains the old instructions instead of the "Notes".

Reply:
The difference between the right version of the old instruction(remove useless `messaging.cloud.google.com/crd-install=true` selector ) and the new version of the instruction is wether we have `cloud-run-events-pre-install-jobs.yam` or not. I updated one more change for that. Please check if this new change would be better.